gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -680,6 +680,120 @@ gst_tensor_filter_common_free_property (GstTensorFilterPrivate * priv)
   gst_tensors_info_free (&priv->out_config.info);
 }
 
+/**
+ * @brief Get available framework from given file when user selects auto option
+ * @param[in] model_files the prediction model paths
+ * @param[in] num_models the number of model files
+ * @return Automatically detected framework name
```

Comment:
It is better to give more information about the ownership of the returning gchar*.

Reply:
Updated. Please check whether the modified message is enough or not.